e2e/test/config/TestConfiguration.Provisioning.cs

```diff
@@ -1,8 +1,19 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Globalization;
+using System.Net;
+using System.Security.Cryptography;
 using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using Microsoft.Azure.Devices.E2ETests.provisioning;
```

Comment:
Will need to be changed with other casing request.

```suggestion
using Microsoft.Azure.Devices.E2ETests.Provisioning;
```